CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.1.0/)
 ### Features
 ### Enhancements
 ### Bug Fixes
+- Fix alert summary prompt ([#958](https://github.com/opensearch-project/flow-framework/pull/958))
```

Comment:
This is just a documentation correction, we can probably skip the changelog.

Reply:
There is `Changelog Verifier` in CI, it got failure if not updating changelog.

Reply:
Reverted

Reply:
> There is `Changelog Verifier` in CI, it got failure if not updating changelog.

Or if you add the `skip-changelog` tag which I did. 😁